### ISSUE TITLE
app_usage_monitor fails trying to call superclass's init

### ISSUE THREAD

**Issue by sheagcraig** [munki/munki#1183]
Running `/usr/local/munki/app_usage_monitor`

results in
```
Traceback (most recent call last):
  File "/usr/local/munki/app_usage_monitor", line 235, in <module>
    main()
  File "/usr/local/munki/app_usage_monitor", line 224, in main
    notification_handler = NotificationHandler.alloc().init()
  File "/usr/local/munki/app_usage_monitor", line 117, in init
    self = super().init()
AttributeError: 'super' object has no attribute 'init'
sys:1: UninitializedDeallocWarning: leaking an uninitialized object of type NotificationHandler
```

Fortunately this is easy to solve-I have a PR incoming.

**Comment by sheagcraig** [comment]
The clue is from this PyObjC issue: https://github.com/ronaldoussoren/pyobjc/issues/549
and newer PyObjC releases toss a warning if you do this.

**Comment by gregneagle** [comment]
Looks like this could have either been triggered by Munki 6.3.0 or 6.3.2, since both had changes in the PyObjC versions. I'm curious if this has been broken a couple of _weeks_ or for several _months_...

**Comment by gregneagle** [comment]
Fixed by PR #1184
